app/src/main/java/org/oppia/app/activity/InputInteractionViewTestActivity.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.app.activity
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import org.oppia.app.R
+import org.oppia.app.customview.inputInteractionView.NumericInputInteractionView
+
+private const val KEY_DIGIT_ID = "DIGIT_ID"
+
+/** This is a dummy activity to test [NumericInputInteractionView]. */
+class InputInteractionViewTestActivity : AppCompatActivity() {
+  private lateinit var digit: String
+  private lateinit var contentComponent: NumericInputInteractionView
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_numeric_input_interaction_view_test)
+    contentComponent = findViewById(R.id.test_number_input_interaction_view) as NumericInputInteractionView
+    if (savedInstanceState != null) {
+      digit = savedInstanceState.getString(KEY_DIGIT_ID)
+    } else {
+      digit = ""
+    }
+    contentComponent.setText(digit)
+  }
+
+  override fun onSaveInstanceState(outState: Bundle) {
+    super.onSaveInstanceState(outState)
+    outState.putString(KEY_DIGIT_ID, getNumberTextInputText())
+  }
+
+  fun getNumberTextInputText(): String {
```

Comment:
Shouldn't this be returning a double since number input are reals?

Reply:
It's removed it was just for saving state for configuration change